packages/design-system/src/components/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -269,5 +272,151 @@ export const NoResults: Story = {
     clearSearchButton: false,
     textFieldLabel: 'This will show a "no results" message.',
     textFieldHint: 'Start typing, but you’ll only get a "no results" message.',
-  } as any,
+  } as AutocompleteArgs,
+};
+
+type MockedDataResponse = {
+  id: number;
+  title: string;
+  artist_title: string;
+};
+const mockedData = [
+  {
+    artist_title: 'Vasily Kandinsky',
+    id: 8980,
+    title: 'Landscape with Two Poplars',
+  },
+  {
+    artist_title: 'Katsushika Hokusai',
+    id: 24645,
+    title:
+      'Under the Wave off Kanagawa (Kanagawa oki nami ura), also known as The Great Wave, from the series "Thirty-Six Views of Mount Fuji (Fugaku sanjūrokkei)"',
+  },
+  {
+    artist_title: 'Eleanor Coen',
+    id: 7030,
+    title: 'Growing City',
+  },
+  {
+    artist_title: 'Amédée Ozenfant',
+    id: 68407,
+    title: 'Landscape (Bordeaux II)',
+  },
+  {
+    artist_title: 'Claude Monet',
+    id: 4783,
+    title: 'Poppy Field (Giverny)',
+  },
+  {
+    artist_title: 'Kurt Seligmann',
+    id: 62323,
+    title: 'Magnetic Mountain',
+  },
+  {
+    artist_title: 'Chaim Soutine',
+    id: 59967,
+    title: 'Landscape at Cagnes',
+  },
+  {
+    artist_title: 'Gustave Moreau',
+    id: 20579,
+    title: 'Hercules and the Lernaean Hydra',
+  },
+  {
+    artist_title: 'Frederic Edwin Church',
+    id: 76571,
+    title: 'View of Cotopaxi',
+  },
+  {
+    artist_title: 'Albert Bierstadt',
+    id: 146701,
+    title: 'Mountain Brook',
+  },
+];
+
+const searchMock = {
+  matcher: {
+    name: 'artworkSearchSuccess',
+    url: 'begin:https://api.artic.edu/api/v1/artworks/search',
+    method: 'GET',
+  },
+  response: {
+    status: 200,
+    body: {
+      data: mockedData,
+    },
+  },
+};
+
+export const AsyncItems: Story = {
+  render: function Component(
+    args: AutocompleteProps & { textFieldLabel: string; textFieldHint: string }
+  ) {
+    const { items, textFieldLabel, textFieldHint, label, ...autocompleteArgs } = args;
+    const [input, setInput] = useState('');
+    const [additionalItems, setAdditionalItems] = useState<AutocompleteItem[]>([]);
+    const hasResults = input.length > 2 && additionalItems.length;
+
+    const searchArtwork = () => {
+      // Note: the response is mocked
+      const searchURL = `https://api.artic.edu/api/v1/artworks/search?q=${input}&fields=id,title,artist_display&limit=10`;
+      fetch(searchURL)
+        .then((response) => {
+          if (response.status === 200) {
+            return response.json();
+          }
+        })
+        .then(({ data }: { data: MockedDataResponse[] }) => {
+          const additionalItems = data.map(({ id, title, artist_title }) => ({
+            id: id.toString(),
+            name: `${title} by ${artist_title}`,
```

Comment:
`artist_title` is coming back undefined. :\

Reply:
I pulled down the branch and added some logging. It looks like each item in the `data` array has an `artist_display` property instead of `artist_title`. Adjusting this should fix the issue.

Reply:
This shouldn't matter if the mock was working correctly. Did y'all run `yarn install` and double check that the network request tab isn't hitting the endpoint?